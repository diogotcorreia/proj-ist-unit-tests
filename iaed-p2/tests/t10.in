set /khS cR`
set /khS/FM4 oQJ
set /khS/FM4/fkq <N[
set /khS/FM4/fkq/z2I 4JT
set /khS/FM4/fkq/OVq 6m[
set /khS/FM4/fkq/lGs ")`
set /khS/FM4/Emx 6|M
set /khS/FM4/Emx/Jz9 sN\
set /khS/FM4/Emx/49T a_]
set /khS/FM4/Emx/i3i ztt
set /khS/FM4/F5S #bk
set /khS/FM4/F5S/hir ?a<
set /khS/FM4/F5S/GbM kYl
set /khS/FM4/F5S/Qt7 #{h
set /khS/XPE l=3
set /khS/XPE/M7R Fs/
set /khS/XPE/M7R/wwz ~1/
set /khS/XPE/M7R/Sj2 f{0
set /khS/XPE/M7R/gpi KN=
set /khS/XPE/KJx V'S
set /khS/XPE/KJx/Ys4 0~n
set /khS/XPE/KJx/xRO nZ]
set /khS/XPE/KJx/ety pu=
set /khS/XPE/n6b 5$[
set /khS/XPE/n6b/RNI *#(
set /khS/XPE/n6b/bMN xLV
set /khS/XPE/n6b/k7S ?tG
set /khS/nTG C>'
set /khS/nTG/cUJ :V,
set /khS/nTG/cUJ/Cxm elM
set /khS/nTG/cUJ/HCP 5E|
set /khS/nTG/cUJ/C_X Dou
set /khS/nTG/KkQ -IS
set /khS/nTG/KkQ/iYs ,(`
set /khS/nTG/KkQ/Jod )i,
set /khS/nTG/KkQ/ZKi f:x
set /khS/nTG/8FB o[N
set /khS/nTG/8FB/gGF aMK
set /khS/nTG/8FB/JAf =6T
set /khS/nTG/8FB/qnu mz%
set /O8O 0yo
set /O8O/Kn_ b~(
set /O8O/Kn_/n5V ]GB
set /O8O/Kn_/n5V/c0I ukg
set /O8O/Kn_/n5V/y1F ro{
set /O8O/Kn_/n5V/2td A_(
set /O8O/Kn_/h1J Mh=
set /O8O/Kn_/h1J/Pho NIN
set /O8O/Kn_/h1J/3HG ';^
set /O8O/Kn_/h1J/GdL 'a/
set /O8O/Kn_/PJ0 thq
set /O8O/Kn_/PJ0/UEp bkk
set /O8O/Kn_/PJ0/ld4 -ra
set /O8O/Kn_/PJ0/rV7 2NQ
set /O8O/VYf k:+
set /O8O/VYf/kGL C}>
set /O8O/VYf/kGL/D55 OJ+
set /O8O/VYf/kGL/jG7 $1N
set /O8O/VYf/kGL/x6F pD:
set /O8O/VYf/Yr5 7g\
set /O8O/VYf/Yr5/Uqj Iqg
set /O8O/VYf/Yr5/Bx1 }?]
set /O8O/VYf/Yr5/Hqn K,(
set /O8O/VYf/nzD m(1
set /O8O/VYf/nzD/JRL KzF
set /O8O/VYf/nzD/ZzJ ]0|
set /O8O/VYf/nzD/mL_ OsG
set /O8O/6IV .pm
set /O8O/6IV/ZYk P4X
set /O8O/6IV/ZYk/K6Q %*`
set /O8O/6IV/ZYk/C28 y*&
set /O8O/6IV/ZYk/Lsw )qd
set /O8O/6IV/E2l eMx
set /O8O/6IV/E2l/oqd o&V
set /O8O/6IV/E2l/HCN ^:&
set /O8O/6IV/E2l/ja6 S/f
set /O8O/6IV/7H5 '(q
set /O8O/6IV/7H5/ke4 ];<
set /O8O/6IV/7H5/YC4 G,d
set /O8O/6IV/7H5/hQK QI+
set /fOy U":
set /fOy/9q1 Y_P
set /fOy/9q1/fZo ,IT
set /fOy/9q1/fZo/Yz9 14"
set /fOy/9q1/fZo/haV ^5o
set /fOy/9q1/fZo/qjs PtB
set /fOy/9q1/545 iS<
set /fOy/9q1/545/4gk Wtz
set /fOy/9q1/545/j0Y `.y
set /fOy/9q1/545/y1W _Ub
set /fOy/9q1/B6e mT8
set /fOy/9q1/B6e/KN9 %yx
set /fOy/9q1/B6e/9WI G<'
set /fOy/9q1/B6e/bWv { j
set /fOy/oVx %sQ
set /fOy/oVx/vyU V=\
set /fOy/oVx/vyU/5le FW9
set /fOy/oVx/vyU/Jqb s&R
set /fOy/oVx/vyU/zZN j"K
set /fOy/oVx/RXc kVv
set /fOy/oVx/RXc/GX7 w71
set /fOy/oVx/RXc/515 B{2
set /fOy/oVx/RXc/WED CcQ
set /fOy/oVx/oI5 d\4
set /fOy/oVx/oI5/u2C `Sm
set /fOy/oVx/oI5/9pI yW@
set /fOy/oVx/oI5/tGv lY=
set /fOy/N0I S]W
set /fOy/N0I/zu1 K1}
set /fOy/N0I/zu1/cDA 7^4
set /fOy/N0I/zu1/RhC YYH
set /fOy/N0I/zu1/KoQ %;v
set /fOy/N0I/bT7 nUT
set /fOy/N0I/bT7/qEf RB`
set /fOy/N0I/bT7/epR d+d
set /fOy/N0I/bT7/cEe 51#
set /fOy/N0I/uRY DfB
set /fOy/N0I/uRY/_HN 8cb
set /fOy/N0I/uRY/qC2 dh\
set /fOy/N0I/uRY/BkX *#}
list
list /khS
list /khS/FM4
list /khS/FM4/F5S
list /O8O/6IV/E2l
list /fOy/N0I/bT7/epR
quit
