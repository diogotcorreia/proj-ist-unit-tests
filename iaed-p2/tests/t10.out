O8O
fOy
khS
FM4
XPE
nTG
Emx
F5S
fkq
GbM
Qt7
hir
HCN
ja6
oqd
